Makefile

```diff
@@ -2,7 +2,7 @@ COMMIT?=${BUILDCOMMIT}
 VERSION?=${BUILDTAG}
 
 # enable cgo because it's required by OSX keychain library
-CGO_ENABLED=1
+CGO_ENABLED=0
```

Comment:
?

Reply:
Cross chain compilation doesn't work with it enabled.